cmd/kubeadm/app/cmd/config.go

```diff
@@ -176,17 +176,17 @@ func getSupportedComponentConfigAPIObjects() []string {
 }
 
 func getDefaultedInitConfig() (*kubeadmapi.InitConfiguration, error) {
-	return configutil.DefaultedInitConfiguration(&kubeadmapiv1beta2.InitConfiguration{
-		// TODO: Probably move to getDefaultedClusterConfig?
+	initCfg := &kubeadmapiv1beta2.InitConfiguration{
 		LocalAPIEndpoint: kubeadmapiv1beta2.APIEndpoint{AdvertiseAddress: "1.2.3.4"},
-		ClusterConfiguration: kubeadmapiv1beta2.ClusterConfiguration{
-			KubernetesVersion: fmt.Sprintf("v1.%d.0", constants.MinimumControlPlaneVersion.Minor()+1),
-		},
-		BootstrapTokens: []kubeadmapiv1beta2.BootstrapToken{placeholderToken},
+		BootstrapTokens:  []kubeadmapiv1beta2.BootstrapToken{placeholderToken},
 		NodeRegistration: kubeadmapiv1beta2.NodeRegistrationOptions{
 			CRISocket: constants.DefaultDockerCRISocket, // avoid CRI detection
 		},
-	})
+	}
+	clusterCfg := &kubeadmapiv1beta2.ClusterConfiguration{
+		KubernetesVersion: fmt.Sprintf("v1.%d.0", constants.MinimumControlPlaneVersion.Minor()+1),
```

Comment:
should be the same as:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/constants/constants.go#L387